x-pack/plugins/triggers_actions_ui/public/application/components/builtin_action_types/slack.tsx

```diff
@@ -86,15 +84,17 @@ const SlackActionFields: React.FunctionComponent<ActionConnectorFieldsProps<
         id="webhookUrl"
         fullWidth
         helpText={
-          <EuiLink
-            href={`${docLinks.ELASTIC_WEBSITE_URL}guide/en/kibana/${docLinks.DOC_LINK_VERSION}/slack-action-type.html`}
-            target="_blank"
-          >
-            <FormattedMessage
-              id="xpack.triggersActionsUI.components.builtinActionTypes.slackAction.webhookUrlHelpLabel"
-              defaultMessage="Create a Slack webhook URL"
-            />
-          </EuiLink>
+          docLinks ? (
```

Comment:
I don't think there would be a scenario where `docLinks` isn't passed in? I think we should only support them being passed in.